builds.json

```diff
@@ -147,11 +147,16 @@
         "name": "nightly user",
         "description": "B2G Nightly user build for Fairphone 2",
         "url": "https://github.com/atilag/b2g-installer-builds/releases/download/b2g-fp2-nightly/FP2.blobfree-dist.zip"
+      },
+     {
+        "name": "B2G OS transition",
+        "description": "Community build from the kanikani and pine transition branches.",
+        "url": "https://www.dropbox.com/s/v6kt9o7aglvxfya/FP2.blobfree-dist.zip?dl=0"
       }
     ],
     "adb": {
       "ro.product.device": [ "FP2" ],
-      "ro.build.display.id": [ "r4275.1_FP2_gms43_1.2.8" ]
+      "ro.build.display.id": [ "r4275.1_FP2_gms43_1.2.8", "r4275.1_FP2_gms54_1.3.6", "r4275.1_FP2_gms56_1.4.2", "r4275.1_FP2_gms59_1.5.1" ]
```

Comment:
I really think we should just have "r4275.1_FP2_gms59_1.5.1" and ask people to perform stock upgrades before: they can contain important low-level fixes that we might need.